SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
I think I found the misunderstanding ☺️

More important log levels should always be logged. In that way you can reduce the noise by less important log levels during release for example. And you can log more by adding filters.

So it should be:

- logLevel >= minLevel = log can happen but just if the exclude filter is not met
- logLevel < minLevel = no logging if no filter is met